src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -148,10 +148,16 @@ if (TARGET mfem)
     # Note - white238: I can't seem to get this to pass install testing due to mfem being included
     # in multiple export sets
```

Comment:
Does the current PR address this comment?

Reply:
This was definitely true at one point. I might have fixed it in the various multiple fixes I've done. Soon I will get back to revisiting all these nightmares and do a bulk review.